RecoTracker/LSTCore/interface/alpaka/Constants.h

```diff
@@ -104,54 +104,80 @@ namespace SDL {
   constexpr float PT_CUT = 0.8f;
 #endif
 
-  const unsigned int MAX_BLOCKS = 80;
-  const unsigned int MAX_CONNECTED_MODULES = 40;
+  constexpr unsigned int MAX_BLOCKS = 80;
+  constexpr unsigned int MAX_CONNECTED_MODULES = 40;
 
-  const unsigned int N_MAX_PIXEL_SEGMENTS_PER_MODULE = 50000;
+  constexpr unsigned int N_MAX_PIXEL_SEGMENTS_PER_MODULE = 50000;
 
-  const unsigned int N_MAX_PIXEL_MD_PER_MODULES = 2 * N_MAX_PIXEL_SEGMENTS_PER_MODULE;
+  constexpr unsigned int N_MAX_PIXEL_MD_PER_MODULES = 2 * N_MAX_PIXEL_SEGMENTS_PER_MODULE;
 
-  const unsigned int N_MAX_PIXEL_TRIPLETS = 5000;
-  const unsigned int N_MAX_PIXEL_QUINTUPLETS = 15000;
+  constexpr unsigned int N_MAX_PIXEL_TRIPLETS = 5000;
+  constexpr unsigned int N_MAX_PIXEL_QUINTUPLETS = 15000;
 
-  const unsigned int N_MAX_PIXEL_TRACK_CANDIDATES = 30000;
-  const unsigned int N_MAX_NONPIXEL_TRACK_CANDIDATES = 1000;
+  constexpr unsigned int N_MAX_PIXEL_TRACK_CANDIDATES = 30000;
+  constexpr unsigned int N_MAX_NONPIXEL_TRACK_CANDIDATES = 1000;
 
-  const unsigned int size_superbins = 45000;
+  constexpr unsigned int size_superbins = 45000;
 
   //defining the constant host device variables right up here
-  ALPAKA_STATIC_ACC_MEM_GLOBAL const float miniMulsPtScaleBarrel[6] = {0.0052, 0.0038, 0.0034, 0.0034, 0.0032, 0.0034};
-  ALPAKA_STATIC_ACC_MEM_GLOBAL const float miniMulsPtScaleEndcap[5] = {0.006, 0.006, 0.006, 0.006, 0.006};
-  ALPAKA_STATIC_ACC_MEM_GLOBAL const float miniRminMeanBarrel[6] = {
+  namespace objLayers {
+    ALPAKA_STATIC_ACC_MEM_GLOBAL constexpr int kpLS =
+        2;  // Currently pixel tracks treated as LSs with 2 double layers (IT layers 1+2 and 3+4). To be potentially handled better in the future.
+    ALPAKA_STATIC_ACC_MEM_GLOBAL constexpr int kLS = 2;
```

Comment:
I think what they meant was that we should remove `ALPAKA_STATIC_ACC_MEM_GLOBAL` and just have it as a `constexpr`. I think we're using this macro wrong anyway since we would have to get that data using `foo<TAcc>.get()`.

(See https://alpaka-group.github.io/alpaka/core_2Common_8hpp.html#a95bebbbf9aab1a18a278359a85a41125)

Reply:
(Didn't realize Slava made the same comment a minute before)

Reply:
Removing the `ALPAKA_STATIC_ACC_MEM_GLOBAL` gives multiple errors like:
```
src/RecoTracker/LSTCore/src/alpaka/MiniDoublet.h(352): error: identifier "SDL::sinAlphaMax" is undefined in device code
```
I will leave it in for now.